cluster-setup/ci-cluster/feature_sriov/sriov-subscription.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sriov-network-operator-subscription
+  namespace: openshift-sriov-network-operator
+spec:
+  channel: "4.3"
```

Comment:
This file should be removed in favour of https://github.com/openshift-kni/performance-addon-operators/pull/28/files#diff-b4137bb24933e2f96ff56df5649dcc80 I think?